packages/frontend/model/extract-capi.ts

```diff
@@ -158,11 +140,8 @@ export const extract = (data: {}): CAPIType => {
     const editionLongForm = getString(data, 'config.page.edition', '');
 
     // Possible values for the editionId: "UK", "US", "AU", "INT"
-    const editionId = getEditionValue(
-        getString(data, 'config.page.editionId', ''),
-    );
-
-    if (editionId === undefined) throw new Error('edition id is undefined');
+    const editionId =
+        getEditionValue(getString(data, 'config.page.editionId', '')) || 'UK';
```

Comment:
Hmm, ok, so the default come from here. Also, should that default be `UK`, I feel like it's maybe `INT` in Frontend (But actually UK makes sense to me, just be good to know that's defo what we want)

Reply:
I was respecting the previously set default here. But you're right that the choice should be deliberate.